### DISCUSSION TITLE
Calculate value after field change (afterStateUpdated)

### DISCUSSION THREAD

**Discussion by OvalMedia** [filamentphp/filament#4811] [Help] [answered]
I have an invoice (with field total) and its children invoiceitems (with fields quantity and price).
Whenever the fields quantity or price of an invoiceitem change I would like to caculate a total and save it in the invoices total field.

The part I am missing here is: how can I access all the other invoiceitems in afterStateUpdated? $get only works in the context of the current item. I cannot access any field of the parent (invoice). Or can I?

Thank you

**Comment by danharrin** [comment, marked as answer]
try `$get('../../repeater_name')`

**Reply by OvalMedia** [reply]
Perfect, thx.
